src/Crocodealer/Core/Label.hs

```diff
@@ -3,28 +3,78 @@ module Crocodealer.Core.Label
        , Label (..)
        , Name (..)
        , HexColor (..)
+       , labelRuleCodec
        ) where
 
+import Toml (Codec (..), Key, TomlCodec, (.=))
+import qualified Toml
+
+disum
```

Comment:
Could you please add a `TODO` here saying to remove this combinator from here after it appears in `tomland`?